pagopa/io-app
# feat: [IOBP-782] Add biz events receipt migration banner info

## ⚠️ This PR depends on #6187 ⚠️

## Short description
This PR adds a banner inside the transaction details to inform the user if that transaction doesn't have the PDF available because it has been imported from the PM (the old payment gateway);

## List of changes proposed in this pull request
- Added a banner info inside the transaction details only if the origin of the transaction is `PM` (that means this transaction is imported);
- Hide the `Get the receipt` button inside the transaction details if the origin is `PM`

## How to test
With the dev server, try to edit the origin of a transaction setting the `origin` property to `PM`, you should be able to see the banner info and there must not be visible the CTA to download the receipt;

## Preview

https://github.com/user-attachments/assets/f475f7d9-93f0-475f-971d-6d1373c84a00
